atsign-foundation/at_app_cli
# Fix for flutter installed via brew

**- What I did**

Changed the method of getting the pub cache path.
Solves an issue with flutter installations via brew.

**- How I did it**

Used the pub_cache package from pub.dev.

**- How to verify it**

Pull the branch
Add a temporary `print(hostedPubCachePath);` at pub/bin/src/file/template_manager.dart:22

```
flutter pub global activate -s path ./pub
flutter pub global run at_app create testing
```

Verify that the pub cache path got printed (should be in the first few lines printed).

**- Description for the changelog**
Reliability for various flutter installation methods.
